docfx_yaml/extension.py

```diff
@@ -1423,11 +1423,32 @@ def find_markdown_pages(app, outdir):
 def convert_cross_references(content: str, current_name: str, entry_names: List[str]):
     words = content.split(" ")
     new_words = []
+
+    # TODO(https://github.com/googleapis/sphinx-docfx-yaml/issues/208):
+    # remove this in the future.
+    iam_policy_link = "http://github.com/googleapis/python-grpc-google-iam-v1/blob/main/google/iam/v1/iam_policy_pb2_grpc.py"
+    hard_coded_references = {
+        "google.iam.v1.iam_policy_pb2.SetIamPolicyRequest": iam_policy_link + "#L103-L109",
```

Comment:
We should probably do some sort of query (e.g. https://github.com/googleapis/python-grpc-google-iam-v1/search?q=setiampolicy+path%3Agoogle%2Fiam%2Fv1%2F+filename%3Aiam_policy_pb2_grpc.py) or use a permalink (e.g. https://github.com/googleapis/python-grpc-google-iam-v1/blob/8e73b45993f030f521c0169b380d0fbafe66630b/google/iam/v1/iam_policy_pb2_grpc.py#L103).